src/coreclr/src/gc/gc.cpp

```diff
@@ -2427,6 +2431,9 @@ static static_data static_data_table[latency_level_last - latency_level_first +
         // gen2
         {256*1024, SSIZE_T_MAX, 200000, 0.25f, 1.2f, 1.8f, 100000, 100},
         // loh
+        {3*1024*1024, SSIZE_T_MAX, 0, 0.0f, 1.25f, 4.5f, 0, 0},
+        // loh
```

Comment:
Should this be `poh`?

Reply:
yes, should be `poh`. 
Initial tuning is copied from `loh` and may be adjusted further, if needed. The comment should say `poh` though.